cmd/alert.go

```diff
@@ -65,6 +83,11 @@ inactive = 0`,
 		// Get all rules from all groups into a single list
 		rules := alert.FlattenRules(alerts.Groups)
 
+		// If there are no rules we can exit early
+		if len(rules) == 0 {
```

Comment:
After looking at the table I created, when --name is defined we should exit UNKNOWN here.

Since the user is expecting the state of a certain alert and it that is not present it might be noteworthy.